cfpb/capital-framework
# Use CFPB brand colors in cf-expandables component

Adds CFPB brand colors as the default variable values for colors in `cf-expandables`. 

Depends on #561 - this includes changes from that branch in the diff.

## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-expandables/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-expandables/

## Screenshots

### before (WITHOUT `cf-theme-overrides.less` colors)

![screen shot 2017-08-18 at 2 24 29 pm](https://user-images.githubusercontent.com/702526/29472762-25812e04-8423-11e7-9b6c-4d353a7970f0.png)
![screen shot 2017-08-18 at 2 24 36 pm](https://user-images.githubusercontent.com/702526/29472775-2ef95e3e-8423-11e7-9809-e90c60ab4232.png)


### after (WITHOUT overrides stylesheet + with brand colors) 

note that fonts aren't loading locally so it looks slightly different from the docs:
![screen shot 2017-08-18 at 2 36 17 pm](https://user-images.githubusercontent.com/702526/29472781-32b0e1fa-8423-11e7-8d14-8d9a415a3195.png)
![screen shot 2017-08-18 at 2 36 25 pm](https://user-images.githubusercontent.com/702526/29472786-35f5da50-8423-11e7-973b-00af301d1a73.png)


## Notes

-

## Todos

-

## Checklist
